src/main/kotlin/com/terraformation/backend/config/TerrawareServerConfig.kt

```diff
@@ -243,6 +246,13 @@ class TerrawareServerConfig(
     }
   }
 
+  @ConstructorBinding
+  class MapboxConfig(
+      val apiToken: String? = null,
```

Comment:
would we need a way to override the token from the admin ui - in case token is compromised? i guess we can always re-roll the service.

Reply:
I think restarting the server is fine. Takes a minute or two and we'd want to do it anyway to change the default in the environment variables even if there were a way to do a temporary override at runtime.

Maybe down the road we'll want to move some of these settings to the database or to a configuration store like Consul so we can change them on the fly.